esmvaltool/diag_scripts/zmnam/zmnam_preproc.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+# Note: requires system install of cdo
+
+
+def zmnam_preproc(ifile):
+
+    os.system('cdo -delete,month=2,day=29 ' + ifile + ' tmp_full_da_nl.nc')
```

Comment:
Please make sure you use the work_dir of the script for writing (temporary) files. This writes to the current working directory, so it will fail if you happen to not have right to write there.

Reply:
I see now that you change working directory in the main script, so it works fine in this case. However, I think it would be better to do that inside this function to avoid confusion. For example, you could pass the working directory as an argument.

Reply:
This function is called after going into work_dir, so this should be safe I think.